sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,45 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(symbol, printer=None):
+    """
+    Returns whether ``symbol`` should be dummified.
+
+    Parameter
+    ---------
```

Comment:
here you are using dashes, but for the 'Examples' section you are using equal-signs.